crates/api_models/src/admin.rs

```diff
@@ -115,6 +115,9 @@ pub struct MerchantAccountCreate {
     /// Default payment method collect link config
     #[schema(value_type = Option<BusinessCollectLinkConfig>)]
     pub pm_collect_link_config: Option<BusinessCollectLinkConfig>,
+
+    #[serde(default)]
+    pub is_network_tokenization_enabled: bool,
```

Comment:
Should this config be in `MerchantAccount`? Can it be moved to `BusinessProfile`?